packages/strapi-plugin-settings-manager/admin/src/assets/images/background_input.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should clean this SVG file by using a tool such as [SVGo](https://github.com/svg/svgo)